test/test.js

```diff
@@ -3,11 +3,31 @@
 const chai = require("chai");
 const assert = chai.assert;
 
-const { idFromURL } = require(".");
+const { idFromURL, getShortcutDetails } = require("./..");
 
 const id = "e04c0db9ef974178b60f94518daeb8f2";
 
+const apiMock = require("./apiMock.json");
+const nock = require("nock");
+
+describe("getShortcutDetails", () => {
```

Comment:
Can these tests be more focused on the output of the function? We're writing tests for Shortcuts.js, not the iCloud Shortcuts API.